source/common/stats/thread_local_store.cc

```diff
@@ -940,6 +940,26 @@ const std::string ParentHistogramImpl::bucketSummary() const {
   }
 }
 
+std::string ParentHistogramImpl::computeDisjointBucketSummary() const {
+  if (used()) {
+    std::vector<std::string> bucket_summary;
+    ConstSupportedBuckets& supported_buckets = interval_statistics_.supportedBuckets();
+    const std::vector<uint64_t> disjoint_interval_buckets =
+        interval_statistics_.computeDisjointBuckets();
+    const std::vector<uint64_t> disjoint_cumulative_buckets =
+        cumulative_statistics_.computeDisjointBuckets();
+    bucket_summary.reserve(supported_buckets.size());
```

Comment:
Paranoia nit: ASSERT here that the 3 array lengths are the same, and make the loop go to their min value?

Reply:
Created ASSERT and looped to min value